.travis.yml

```diff
@@ -91,9 +92,11 @@ install:
     - python setup.py build_ext --inplace
     - cd $girder_path
     - pip install -U -r requirements.txt -r requirements-dev.txt setuptools==19.4
-    - pip install -U -r $large_image_path/requirements.txt 
+    - pip install -U -r $large_image_path/requirements.txt
     - pip install -U -r $slicer_cli_web_path/requirements.txt 
     - npm install
+    - cd $large_image_path
+    - python setup.py install
```

Comment:
It shouldn't be.  For large image, this is just an alternate way of installing the requirements.txt list.